src/instructlab/sdg/generate_data.py

```diff
@@ -220,6 +221,25 @@ def _sdg_init(ctx, pipeline):
     data_dirs = [os.path.join(xdg_data_home(), "instructlab", "sdg")]
     data_dirs.extend(os.path.join(dir, "instructlab", "sdg") for dir in xdg_data_dirs())
 
+    docling_model_path = None
+    sdg_models_path = docling_model_path
+    for d in data_dirs:
+        if os.path.exists(os.path.join(d, "models")):
+            sdg_models_path = os.path.join(d, "models")
+            break
+
+        if sdg_models_path is not None:
+            try:
+                with open(
+                    os.path.join(sdg_models_path, "config.yaml"), "r", encoding="utf-8"
+                ) as file:
+                    config = yaml.safe_load(file)
+                    docling_model_path = config["models"][0]["path"]
```

Comment:
It made sense to have it here, but I hear you. Returning more values than this will be too hacky.

Reply:
@bbrowning would you suggest changing this flow and eliminate any dependencies on the generate_data.py and rather have all of this as a part of chunkers.py? Since the `chunk_documents` function is being called as a part of the taxonomy functions it made more sense to have this flow instead.

Reply:
I think if get the indentation fixed and this value returned from the method so that it gets used by where `_sdg_init` is called, we can clean these other things up downstream. I'm not opposed to working on that now, but also more than happy to revisit this once we get the initial bits working.

Reply:
I think by doing this we can eliminate the need of passing this as a parameter to generate_data. We just need this value to be passed through the taxonomy functionalities as well